app/controllers/more-info.js

```diff
@@ -154,6 +154,17 @@ export default Ember.Controller.extend({
     },
     setFrequency: function(frequency){
       this.model.set('update_frequency', frequency);
+    },
+    setRadioButton: function(checkbox) {
+      /*eslint-disable */
```

Comment:
why does this part have to be disabled for linting?

Reply:
It causes an error for CircleCi because it's jquery. Not sure how to go about it.

Reply:
Aha. I see what you are trying to achieve here. I have some thoughts on it. 

1. It might not be a good idea to name class used for radio button to be `checkbox`. This can give a wrong idea about what type of input was used. It is generally a good practice to avoid reserved namespace such as checkbox, checked etc. 

2. Maybe we can rethink about using class name to target the element. Class name is not meant to be used to target the element. This can return the result we want at first, but can make code hard to manage. ex. If we change class name from `checkbox` to something else, or decided to give different style by changing class name, we would have to change the selector here too. 

3.  I will take a look again after  all the changes from master picked up again. However, I wonder this UI trick works for data. (Does this trigger any function which actually sets the data type of submission? == When $('.provide.checkbox') is checked, does submission model pick up this change and change the data type to `provide`? )

Reply:
sorry not sure I'm understanding. Are you talking about when I use .checkbox for jquery or the checkbox name I'm passing as a parameter to the function? 
If it's the first, the .checkbox is used by semantic-ui's radio buttons, so should I add another class in order to target it? If it's the second, I can change it to another parameter name. 

For your third question, I think you are asking if the jquery checkbox thing changes licenseType? If so, then yes it does change the data type to provide.

Reply:
ohhh I think it is from my ignorance of semantic-ui. I thought checkbox is something you decided to name the class, (which will be interesting choice for radio button), but now I see semantic-ui's https://semantic-ui.com/modules/checkbox.html documentation that it can be used for radio button.. I got your point 

I am not sure what would be best approach for 2nd one. because targeting something with class doesn't seem to be a great choice especially inside of Framework, which is supposed to hook up data with matching element so that we can avoid approaching to elements through global context. However, I am also not sure what would be the best approach for this. so for this pr, I would suggest for you to pick up what it should do, and we will pick up how to solve this problem later.

Reply:
Wait, I think I can do it without using jquery. 
Since it's checking `licenseType` and mutating when the radio button is clicked, for the `setRadioButton` function instead of using jquery to check the radio button, I can set `licenseType` to be the correct one and the radio button should be selected. 
I am pushing the changes now. If you don't mind, can you test it out and see if it still picks up the errors?